styles.css

```diff
@@ -1,182 +1,224 @@
 body {
```

Comment:
- [Optional] Dear @Mekuseo, I am only making this optional since it's not part of this review. **But** I strongly suggest you address this. As you can see on the screenshot I attached your design is not responsive on a 375px(minimum) screen size. Please fix this as this will benefit you in the future.

![responsive](https://user-images.githubusercontent.com/69077061/199205230-2eddeb91-a628-4b12-820b-9ff0d8d3659f.PNG)